### PR TITLE
Updated htop-osx to include patch to fix SIGSEGV fault when COLS > 178.

### PR DESCRIPTION
**Pull request by max-horvath**
Closes https://github.com/Homebrew/homebrew/pull/34684 ...

### CODE REVIEW

File: Library/Formula/htop-osx.rb [hunk 1]
```diff
@@ -1,7 +1,9 @@
+require "formula"
+
```
Comment (mikemcquaid) [inline Library/Formula/htop-osx.rb]: You don't need this any more.

Comment (max-horvath) [inline Library/Formula/htop-osx.rb]: Will remove it.

**Comment by max-horvath** [comment]
Should be ready to go.

**Comment by mikemcquaid** [comment]
Thanks for the quick turnaround @max-horvath!

**Pull request closed without merging**